webpack.config.js

```diff
@@ -0,0 +1,60 @@
+const path = require('path');
+const webpack = require('webpack');
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
+const packageData = require('./package.json');
+
+
+module.exports = {
+  entry: {
+    admin: ['./admin/assets/admin.js', './admin/assets/admin.css'],
+    vendor: ['semantic-ui-css', 'semantic-ui-css/semantic.css', 'jquery']
+  },
+  output: {
+    filename: '[name].bundle.min.js',
+    path: path.resolve(__dirname, 'dist/admin/assets')
+  },
+  devtool: 'source-map',
```

Comment:
FM course tip: Consider using 'eval' for development and 'source-map' for prod builds. 'eval' embeds the maps on the file, which (supposedly) makes it easier to debug.
ex: devtool: process.env.NODE_ENV === 'production' ? 'source-map' : 'eval'